joss/paper.md

```diff
@@ -127,12 +129,12 @@ Python vignettes that illustrate their use. While the origins of
 `fitgrid` are in EEG data analysis, `fitgrid` can also be used with
 other neuroimaging data such as MEG and more generally with
 synchronized sensor array time-series data from other domains for
-which event-related regression modeling is appropriate.  `fitgrid`
+which event-related regression modeling is appropriate. `fitgrid`
 enables researchers to conduct this type of computationally-intensive
 modeling flexibly, efficiently, informatively, and reproducibly with
 familiar scientific computing tools and minimal programming. These
 features make `fitgrid` well-suited for general use in exploratory
-data analysis (EDA), e.g., @UrbDelChaKut2020 and @TroUrbKut2020.
+data analysis (EDA; e.g., @UrbDelChaKut2020 and @TroUrbKut2020).
 
 ![fitgrid TL; DR](fitgrid_overview.png)
```

Comment:
I would prefer you add a proper figure caption here. I am not sure in how far the format permits that ... but the "TL; DR" is a bit too brief.

Reply:
Done